CET-8301 Update backstage after initiative filter changes (#164)
cortexapps/backstage-plugin: CHANGELOG.md

# Changelog

### 2.6.5

> - Fixes issue where groups filters were not rendered correctly in metadata on Initiative page

Reviewer: Sigh look what happens when I'm gone
```suggestion
- Fix issue where groups filters are not rendered correctly in metadata on Initiative page
```

Please apply tense fix to the 3 changelog entries below this as well
